src/NeoSharp.Application/Client/Prompt[Wallet].cs

```diff
@@ -10,70 +10,47 @@ public partial class Prompt : IPrompt
         [PromptCommand("wallet create", Category = "Wallet", Help = "Create a new wallet")]
         private void WalletCreateCommand(string fileName)
         {
-            try
-            {
-                _walletManager.CreateWallet(fileName);
-                SecureString secureString = _consoleReader.ReadPassword();
-                _walletManager.CreateAccount(secureString);
-            }
-            catch(System.Exception exception)
-            {
-                _consoleWriter.WriteLine(exception.Message, ConsoleOutputStyle.Error);
-            }
-
+            _walletManager.CreateWallet(fileName);
+            SecureString secureString = _consoleReader.ReadPassword();
+            _walletManager.CreateAccount(secureString);
         }
 
         [PromptCommand("wallet open", Category = "Wallet", Help = "Open wallet")]
         private void WalletOpenCommand(string fileName)
         {
-            try
-            {
-                _walletManager.Load(fileName);
-            }catch (System.Exception exception)
-            {
-                _consoleWriter.WriteLine(exception.Message, ConsoleOutputStyle.Error);
-            }
+            _walletManager.Load(fileName);
         }
 
         [PromptCommand("wallet close", Category = "Wallet", Help = "Close wallet")]
         private void WalletCloseCommand()
         {
-            try{
-                _walletManager.Close();
-            }catch (System.Exception exception)
-            {
-                _consoleWriter.WriteLine(exception.Message, ConsoleOutputStyle.Error);
-            }
+            _walletManager.Close();
```

Comment:
_walletManager?.Close();

Reply:
@shargon _walletManager is injected, do we really need to check if it's null in all usages?